### ISSUE TITLE
using the "exclude from purge" array when constructing ORMPurger

### ISSUE THREAD

**Issue by patrickmyh** [liip/LiipFunctionalTestBundle#289]
Hey there,

for some time already (july 2016) , doctrine/data-fixtures' ORMPurger offers a nice solution to exlude certain tables from being purged when loading the fixtures. These are being passed in as a constructor argument.

I'd like to use this in your bundle as well, but unfortunately liip's WebTestCase loadFixtures method constructs the ORMPurger without any further arguments 

```php
if ('PHPCR' === $type) {
    $purger = new $purgerClass($om);
    [...]
    $executor = new $executorClass($om, $purger, $initManager);
} else {
    $purger = new $purgerClass();
    [...]
    $executor = new $executorClass($om, $purger);
}
```
So I thought about adding a property in WebTestCase containing the array with tablesnames, to be set from the inheriting class and used there. 

I'll add a pull request if that's the right approach in your eyes.

**Comment by lsmith77** [comment]
yeah feel free to send a PR ..

in general the WebTestCase class should one day get split up more to use "services" but until then your approach is likely the right way.

**Comment by alexislefebvre** [comment]
It has been fixed in #298.
